lib/create-organization-api.js

```diff
@@ -193,6 +194,50 @@ export default function createOrganizationApi ({ http }) {
       .then(response => wrapTeamMembershipCollection(http, response.data), errorHandler)
   }
 
+   /**
+   * Get a Team Space Membership with given teamSpaceMembershipId
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} teamSpaceMembershipId - Team ID
+   * @param {string} id - Team Membership ID
```

Comment:
i think we do not need line 201 here and in 200 we should either describe it as Team Space Membership Id or just remove the Team ID part since probably the argument is specific enough

Reply:
Busted!